jetty-core/jetty-io/src/main/java/org/eclipse/jetty/io/Content.java

```diff
@@ -562,19 +663,31 @@ static void write(Sink sink, boolean last, String utf8Content, Callback callback
      * to release the {@code ByteBuffer} back into a pool), or the
      * {@link #release()} method overridden.</p>
      */
-    public interface Chunk extends Retainable
+    public interface Chunk extends RetainableByteBuffer
```

Comment:
I think I'm at peace with the idea that a Chunk is a RBB, despite it happening on line 666.